pkg/handler/main.go

```diff
@@ -19,14 +20,14 @@ func OnMessageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 	// we should parse the link out of m.Content
 	re := regexp.MustCompile(`^http*\.*`)
 	if re.MatchString(m.Content) {
-		short := api.Bitly(m.Content)
-		if short == "error" {
+		short, err := bitly.Shorten(m.Content)
+		if err != nil {
+			log.Errorf("creating short link %v\n", err)
```

Comment:
Bubbling up errors that happen in `pkg/bitly` or other places is a good idea as well as logging them from here.
Since we can return multiple values from a function, we can change the method signature of bitly.Shorten to return both the shortened string and an error if there is any.

Reply:
ahh, I get that. Is there a reason to not log at the spot it occurs?

Reply:
There's nothing inherently bad about it. But as the code base increases in size, it's sometimes hard to figure out exactly where something is logging from.